src/openzaak/selectielijst/admin_fields.py

```diff
@@ -84,10 +86,16 @@ def get_processtype_readonly_field(url: str) -> str:
 
 
 def get_resultaat_readonly_field(url: str) -> str:
-    resultaat = retrieve_resultaat(url)
-    return f"{resultaat['volledigNummer']} - {resultaat['naam']} - {resultaat['waardering']}"
+    try:
+        resultaat = retrieve_resultaat(url)
+        return f"{resultaat['volledigNummer']} - {resultaat['naam']} - {resultaat['waardering']}"
+    except InvalidURLError:
+        return "De selectielijstklasse is niet afkomstig uit de Referentielijsten API-service"
```

Comment:
Let's make this message translatable
```suggestion
        from django.utils.translations import gettext_lazy as _
        ...
        return _("De selectielijstklasse is niet afkomstig uit de Selectielijst API-service")
```